.circleci/config.yml

```diff
@@ -143,29 +143,31 @@ jobs:
       - *restore-gems
       - *install-gems
       - *store-gems
-      - run:
-          name: Download podspec repo
-          command: curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
+      - restore_cache:
+          keys:
+            - cocoapods-{{ checksum "Podfile.lock" }}
+            - cocoapods-
       - run:
           name: Lint podspec
           command: bundle exec rake pod:lint
+      -  save_cache:
+          key: cocoapods-{{ checksum "Podfile.lock" }}
+          paths:
+            - ~/.cocoapods
```

Comment:
With CDN, is it really still worth it to cache the `~/.cocoapods` folder? From experience on other repos it doesn't really seem to be much necessary anymore.

We already commit the `Pods/` folder (and if we didn't, we could cache it) so it should be quite rare to have the need for the spec repo to be present and downloaded locally anyway, as it is now supposed to only be necessary during `pod update` (or if `pod install` needs to actually install pods that were missing in `Pods/` and were not at a version that was matching was was locked in `Podfile.lock`, which in our case should never happen on CI)

Reply:
iow, if you remove those steps from CI, and as long as we still commit the `Pods/` folder and that `Podfile.lock` and `Pods/Manifest.lock` are properly in sync, I think `pod install` is now a no-op that shouldn't even require the CDN to be cloned at all.
Maybe I'm wrong so don't hesitate to make some tests to validate that if unsure.